src/packagedcode/pypi.py

```diff
@@ -83,8 +84,8 @@ def logger_debug(*args):
 
 @attr.s()
 class PythonPackage(models.Package):
-    metafiles = ('metadata.json', '*setup.py', 'PKG-INFO', '*.whl', '*.egg')
-    extensions = ('.egg', '.whl', '.pyz', '.pex',)
+    metafiles = ('metadata.json', '*setup.py', 'PKG-INFO', '*.whl', '*.egg', '*.txt')
+    extensions = ('.egg', '.whl', '.pyz', '.pex', '.txt')
```

Comment:
Not sure we want to have *.txt or .txt here. This will classify text files as requirements files, right?

Reply:
The main naming conventions I'm aware of are `requirements.txt`, `dev-requirements.txt` and `locked-requirements.txt` (I'm not sure how widespread the last one is, I just prefer using "requirements.txt" and "locked-requirements.txt" when using pip-compile)

Reply:
@ncoghlan @majurg @pombredanne filename can be `requirements.txt`, `dev-requirements.txt`, `test-requirements.txt`, `requirements-dev.txt` or `requirements-test.txt` so what should I choose??

Reply:
@rpotter12 IMHO all of them :) ... but what this really means is that the function that detects and parses requirements need to be flexible there and may not rely too much on the generic package type metadata.